cartridge.lua

```diff
@@ -804,6 +805,25 @@ local function cfg(opts, box_opts)
     if type(box.cfg) == 'function' then
         confapplier.log_bootinfo()
     end
+    local crg_opts_to_logs = table.deepcopy(opts)
+    crg_opts_to_logs.cluster_cookie = nil -- remove cluster_cookie from logs
+
+    local box_opts_to_logs = table.deepcopy(box_opts)
+    -- remove password and login from logs:
+
+    if type(box_opts_to_logs.replication) == 'string' then
+        box_opts_to_logs.replication = { box_opts_to_logs.replication }
+    end
+
+    local replication = {}
+    for _, v in ipairs(box_opts_to_logs.replication or {}) do
+        local uri = v:split('@')[2]
```

Comment:
Is it possible to use "@" in password? Probably uri.parse is a bit safer for such purpose.